config/config.toml.example

```diff
@@ -124,6 +124,8 @@ metrics-interval = 15
 max-procs = 0
 # StmtCountLimit limits the max count of statement inside a transaction.
 stmt-count-limit = 5000
+# If true, the transaction will be committed when it reaches stmt-count-limit and starts a new transaction.
+batch-commit = false
```

Comment:
Too many configure options will make the system unfriendly to the user.
How about reusing the global session variable batch commit ?

Reply:
We don't have a global batch commit variable. There are two session variables batch_insert and batch_delete. This config should only be used when loading data. I think it is not safe to set it to true, so keep it unfriendly. And it may be dangerous to make it as a global variable.